src/redux/slices/bookingFlightsSlice.js

```diff
@@ -1,42 +1,72 @@
-import { createSlice, createAsyncThunk } from '@reduxjs/toolkit'
+import { createSlice, createAsyncThunk, current } from '@reduxjs/toolkit'
 import flightAPI from '../../api/Flight'
 import discountInfo from '../../api/Discount'
+import moment from 'moment'
 const initialState = {
   loadding: false,
   userInformation: {},
   dataFlight: {},
   discountInfo: {
     percent: 0,
   },
+  priceAfterDiscount: 0,
 }
 export const getDataFlights = createAsyncThunk(
   'flight/getDataFlights',
   async (idFlight) => {
-    const respone = await flightAPI.get(1)
+    const respone = await flightAPI.get(idFlight)
     return respone.data
   }
 )
 
 export const getDiscountCheck = createAsyncThunk(
-  'discount/getDiscountCheck',
+  'flight/getDiscountCheck',
   async (idFlight) => {
     const respone = await discountInfo.getDiscountById(idFlight.idDiscount)
     return respone.data
   }
 )
+export const getUserDataInBooking = createAsyncThunk(
+  'flight/getUserData',
+  async (idUser) => {
+    const respone = await flightAPI.getUserData(idUser)
+    return respone.data
+  }
+)
+
+export const createBookingFlight = createAsyncThunk(
+  'flight/createBooking',
+  async (params) => {
+    const respone = await flightAPI.createATicket(params)
+    return respone.data
+  }
+)
+
 const bookingFlightsSlice = createSlice({
   name: 'filterSlice',
```

Comment:
I don't know why bookingFlightsSlice is named filterSlice. Is it a mistake ? 😁

Reply:
He copy old code but forgot change their name :))